cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-upgrading.adoc

```diff
@@ -0,0 +1,78 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-getting-started-upgrading"]
+= Tutorial: Upgrading your cluster
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-getting-started-upgrading
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-01-08
+
+{product-title} (ROSA) executes all cluster updates as part of the managed service. You do not need to run any commands or make changes to the cluster. You can schedule the upgrades at a convenient time.
+
+Ways to schedule a cluster upgrade include:
+
+* *Manually using the command line interface (CLI)* - Start a one-time immediate upgrade or schedule a one-time upgrade for a future date and time.
+* *Manually using the Red Hat OpenShift Cluster Manager user interface (UI)* - Start a one-time immediate upgrade or schedule a one-time upgrade for a future date and time.
+* *Automated upgrades* - Set an upgrade window for recurring y-stream upgrades whenever a new version is available without needing to manually schedule it. Minor versions have to be manually scheduled.
+
+For more details about cluster upgrades, run the following command:
+
+[source,terminal]
+----
+$ rosa upgrade cluster --help
+----
+
+== Manually upgrading your cluster using the CLI
+. Check if there is an upgrade available by running the following command:
++
+[source,terminal]
+----
+$ rosa list upgrade -c <cluster-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+$ rosa list upgrade -c <cluster-name>
+VERSION  NOTES
+4.10.20   recommended
+4.10.18
+...
+----
++
+In the above example, versions 4.10.18 and 4.10.20 are both available.
+
+. Schedule the cluster to upgrade within the hour by running the following command:
++
+[source,terminal]
+----
+$ rosa upgrade cluster -c <cluster-name> --version <desired-version>
+----
+
+. *Optional:* Schedule the cluster to upgrade at a later date and time by running the following command:
++
+[source,terminal]
+----
+$ rosa upgrade cluster -c <cluster-name> --version <desired-version> --schedule-date <future-date-for-update> --schedule-time <future-time-for-update>
+----
+
+== Manually upgrading your cluster using the UI
+. Log in to the OpenShift Cluster Manager, and select the cluster you want to upgrade.
+. Click *Settings*.
+. If an upgrade is available, click *Update*.
++
+image::cloud-experts-getting-started-cluster-upgrade.png[]
+
+. Select the version to which you want to upgrade in the new window.
+. Schedule a time for the upgrade or begin it immediately.
+
+== Setting up automatic recurring upgrades
+. Log in to the OpenShift Cluster Manager, and select the cluster you want to upgrade.
+. Click *Update Settings*.
+1. Under *Update Strategy*, click *Automatic*.
```

Comment:
Would you click "Automatic" or "Recurring updates"? I may not be seeing this the way an actual end user would, so not sure.